pastas/project/project.py

```diff
@@ -59,6 +61,10 @@ def __init__(self, name, metadata=None):
             Dictionary with any metadata information on the project.
 
         """
+        warn("The Project class will be deprecated in a future verion of "
+             "Pastas! Consider switching to pastastore: "
```

Comment:
We need to end a time here I think. End of 2020?

Reply:
sounds good